src-electron/util/types.js

```diff
@@ -78,6 +78,78 @@ async function typeSizeAttribute(db, zclPackageId, at, defaultValue = null) {
   }
 }
 
+/**
+ *
+ * @param {*} value
+ * @param {*} size
+ * @returns The negative hex value for a given positive hex value padded with
+ * the given size.
+ */
+function convertFloatToHex(value, size) {
+  let res = ''
+  const arrayBuffer = new ArrayBuffer(size)
+  const dataView = new DataView(arrayBuffer)
+  dataView.setFloat32(0, value, false)
+  for (let i = 0; i < size; i++) {
+    let t = dataView.getUint8(i).toString(16)
+    res += t == 0 ? '00' : t
```

Comment:
Why is this `t == 0` check here?

In any case, I don't think this is right.  If we get a byte with value `1` this will append `"1"` to the string, whereas it should append `"01"`, no?